app/hub.py

```diff
@@ -343,8 +351,12 @@ def send_tx(self, amount, address, payment_id, priority, mixin, tx_desc, save_ad
             return
         
         amount = int(amount*COIN)
-        ret = self.ui.wallet_rpc_manager.rpc_request.transfer_split(amount, \
-                                            address, payment_id, priority, mixin)
+        
+        if sweep_all:
+            ret = elf.ui.wallet_rpc_manager.rpc_request.transfer_all(address, payment_id, priority, mixin)

```

Comment:
Should be:

`self.ui.wallet_rpc_manager.rpc_request...`

Reply:
corect